src/gpl/src/nesterovBase.h

```diff
@@ -1021,47 +1090,13 @@ class NesterovBase
   int64_t stdInstsArea_ = 0;
   int64_t macroInstsArea_ = 0;
 
-  std::vector<GCell> gCellStor_;
-
-  std::vector<GCell*> gCells_;
-  std::vector<GCell*> gCellInsts_;
-  std::vector<GCell*> gCellFillers_;
+  std::set<std::shared_ptr<GCell>, GCellComparator> newGCells_;
```

Comment:
What does 'new' denote?  Should this just be gCells_ ?

Reply:
Yes, I was using 'new' only temporarily, and I forgot to rename it afterward.